example/spec/parallel/ingestGranule/IngestGranuleFailureSpec.js

```diff
@@ -104,11 +104,22 @@ describe('The Ingest Granule failure workflow', () => {
   });
 
   afterAll(async () => {
-    // clean up stack state added by test
-    await deleteGranule({
-      prefix: config.stackName,
-      granuleId: inputPayload.granules[0].granuleId,
-    });
+    // The granule provided cannot clean up the
+    // fake S3 objects, so we expect it to fail
+    try {
+      await deleteGranule({
+        prefix: config.stackName,
+        granuleId: inputPayload.granules[0].granuleId,
+        pRetryOptions: {
```

Comment:
The catch logic in this block should probably be refactored to use the `onFailedAttempt` method parameter so that pRetry *can* retry on failures we're not expecting.

Reply:
Hmm, withdrawn - updating this in that way isn't actually as useful as I thought it was.